src/Elders.Cronus/IConsumer.cs

```diff
@@ -0,0 +1,14 @@
+namespace Elders.Cronus
+{
+    public interface IConsumer<T>
+    {
+        void Start();
+
+        void Stop();
+    }
+
+    interface IInMemoryConsumer<T> : IConsumer<T>
```

Comment:
Why do we need this interface?

Reply:
Probably we do not. This will be resolved when someone handles the in memory implementation and make it working